src/style.css

```diff
@@ -1,3 +1,80 @@
+.todo__section {
```

Comment:
- [ ] Good job so far :clap:  But I think you could improve your design a make it similar to given [video](https://www.youtube.com/watch?v=AcUd-_Yjjqg) . I would recommend you to center the todo list and also apply the border line as you can see in the shared video. :+1: